### DISCUSSION TITLE
My maps are gone AGAIN

### DISCUSSION THREAD

**Discussion by Tremayah** [ppy/osu#25961] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

This happened before multiple times, to the point where I literally uninstalled everything to do with osu. All files, mentions, app data, everything was gone; a complete fresh start. Re install osu, start downloading new maps everything is normal and then in the most recent update I get the same "database is too new for this version, consider downgrading" message and all my maps are gone. Is it something to do with my windows version? Graphics drivers? Account?? I actually can't possibly think of why this is happening after so meticulously deleting everything to do with osu.

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/125309136/b751d09c-349d-491a-a46a-4b424e002281)

### Version

2023.1130.0-lazer

### Logs

Forgot to export logs when the bug happened, sorry!

**Comment by Tremayah** [comment]
Forgot to edit type, apologies.

**Comment by peppy** [comment]
So two things: logs are now stored, so please click "export logs" in settings and attach the zip file.

Secondly, this generally happens if you run a new version then an older one. You can easily resolve this:

- Install the latest version from the website by running the install (DON'T uninstall first)
- Close the game
- Delete `client` (the 64kb file)
- Rename `client_newer_version` to `client` (the 2mb file)

**Reply by Tremayah** [reply, marked as answer]
Simply reinstalling fixed the issue. The issue is actually caused by Geforce Experience launching the old version of the game, regardless of when Osu updates. This is why there would constantly be issues whenever there were updates, as this is how I always launch the game. Geforce Experience sees every update as an entirely new game.
![Uploading image.png…]()

**Reply by peppy** [reply]
Aha, good to know. This will be fixed once we switch to a new version of the updater, hopefully soon.
